### PR TITLE
[Spring Beans]Lps 200088

### PR DESCRIPTION
**Pull request by slnn**
https://liferay.atlassian.net/browse/LPS-200088

### CODE REVIEW

**Comment by slnn** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:upgrade

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by Kvale1733** [comment]
ci:test:upgrade

**Comment by Kvale1733** [comment]
@slnn No regressions found.

File: portal-impl/src/META-INF/portlet-container-spring.xml [hunk 1]
```diff
@@ -36,5 +30,6 @@
 	<bean class="com.liferay.portal.kernel.portlet.PortletURLFactoryUtil" id="com.liferay.portal.kernel.portlet.PortletURLFactoryUtil">
 		<property name="portletURLFactory" ref="com.liferay.portlet.PortletURLFactory" />
 	</bean>
+	<bean class="com.liferay.portlet.PortletContextFactoryUtil" id="com.liferay.portlet.PortletContextFactoryUtil" />
```
Comment (dantewang) [inline portal-impl/src/META-INF/portlet-container-spring.xml:33]: @slnn Since it's already a util, we don't need to add a bean declaration here.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
@slnn Please resend, thanks!

**Comment by slnn** [comment]
Resend in https://github.com/liferay-core-infra/liferay-portal/pull/6001

**Pull request closed without merging**